views/js/qtiCreator/widgets/interactions/extendedTextInteraction/states/Question.js

```diff
@@ -139,6 +141,20 @@ define([
             interaction.attr('patternMask', attrValue);
         };
 
+        callbacks.expectedLength = function(interaction, attrValue){
+            var newValue = parseInt(attrValue,10);
+            if(! isNaN(newValue)){
+                interaction.attr('expectedLength', attrValue);
+            }
+        };
+
+        callbacks.expectedLines = function(interactions, attrValue){
+            var newValue = parseInt(attrValue,10);
+            if(! isNaN(newValue)){
+                interaction.attr('expectedLines', attrValue);
+            }
```

Comment:
Isn't it supposed to have a visual feedback to the user ? like dynamic box resizing along with this attribute modification ? The issue currently is that as an item author, we have no idea of what this option is supposed to do.

As per the standard : 
Attribute : expectedLines [0..1]: integer
The expectedLines attribute provides a hint to the candidate as to the expected number of lines of input required. A line is expected to have about 72 characters. A Delivery Engine should use the value of this attribute to set the size of the response box, where applicable. This is not a validity constraint.